src/components/common/text-area/index.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect, useRef } from "react";
+import PropTypes from "prop-types";
+import { TEXTAREA_DEFAULT_ROW } from "utils/const/formElements";
+
+function TextArea(props) {
+  const { errorMessage, numOfRows, name, shouldAutoFocus, value, onChange } =
+    props;
+  const taskInputRef = useRef(null);
+
+  useEffect(() => {
+    if (shouldAutoFocus) {
+      taskInputRef.current.focus();
+    }
+  }, []);
```

Comment:
If the value of `shouldAutoFocus` changes after the initial render, the effect will not be re-evaluated, hence the autofocus behavior will not be updated.